tests/dataset_runner.py

```diff
@@ -67,6 +67,13 @@ def run(self, dataset_fixture, run_name_prefix="test"):
                 self.retrieve_zarr_output_from_matrix_service()
                 self.retrieve_loom_output_from_matrix_service()
             self.assert_data_browser_bundles()
+
+        for primary_bundle_uuid, secondary_bundle_fqid in self.primary_uuid_to_secondary_bundle_fqid_map.items():
+            self.data_store.tombstone_bundle(primary_bundle_uuid)
+            if secondary_bundle_fqid is not None:
+                secondary_bundle_uuid = secondary_bundle_fqid.split('.')[0]
+                self.data_store.tombstone_bundle(secondary_bundle_uuid)
+
```

Comment:
Better! but:
Please move this code into a private method, called something like `_cleanup_test_result_bundles`.  This will keep `run()` clean - it pretty much only has calls to other methods, and adds a description of what the code actually does, without having to add a comment above it or read all the way through it to understand what it does.

Reply:
If you've never heard of them, Sandi Metz' File Rules keep code readable: https://robots.thoughtbot.com/sandi-metz-rules-for-developers

Reply:
Hey, I'm all for readability. Although the five lines per method rule seems next to impossible :)

Fixed.